Adyen.podspec

```diff
@@ -45,6 +45,18 @@ Pod::Spec.new do |s|
     plugin.dependency 'CashAppPayKitUI', '0.5.1'
   end
 
+  s.subspec 'AdyenTwint' do |plugin|
+    plugin.source_files = 'AdyenTwint/**/*.swift'
+    plugin.dependency 'Adyen/Core'
+    plugin.dependency 'Adyen/Actions'
+    plugin.dependency 'Adyen/TwintSDK'
+  end
+
+  s.subspec 'TwintSDK' do |plugin|
```

Comment:
Can this subspec be put into AdyenTwint subspec to prevent this from being podded directly?